client_cli/pytests/src/client_cli/iroha.py

```diff
@@ -72,13 +77,25 @@ def assets(self) -> List[str]:
         Retrieve assets from the Iroha network and return them as list of ids.
 
         :return:  List of assets ids.
-        :rtype: List[str] 
+        :rtype: List[str]
         """
         self._execute_command('asset')
         assets = json.loads(self.stdout)
         assets = [asset["id"] for asset in assets]
         return assets
 
+    def get_quantity(self, asset_id):
```

Comment:
This function seems error prone. What if latest executed command was not asset.

I would suggest to change `asset` function such that it will return map of full object.

As in case with `asset_definitions`.